tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `release`
+(
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `info` VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
```

Comment:
What is the default charset on our MySQL servers? Shouldn't we make sure that all the fields are UTF8? Or do we not care about that?